hive_mind_hive/app/views/shared/_hive_plugin_partial.html.erb

```diff
@@ -6,16 +6,19 @@
 <div class="row">
   <div id="load_label_<%= device.id -%>" class="col-xs-2"><span class="label label-default">Load</span></div>
   <% ld = device.device_statistics.where(label: 'Load average').last %>
+  <% nproc = device.device_statistics.where(label: 'Processor count').last %>
+  <% nproc = nproc ? nproc.value : 1 %>
   <% chart_svg = "load_chart_#{device.id}" %>
-  <% state = ld ? ( ld.value < 0.7 ? 'success' : ld.value < 1.5 ? 'warning' : 'danger' ) : 'info' %>
-  <div class="col-xs-2"><span class="label label-<%= state -%>"><%= device.latest_stat('Load average', '%.2f') -%></span></div>
+  <% state = ld ? ( ld.value/nproc < 0.7 ? 'success' : ld.value/nproc < 1.5 ? 'warning' : 'danger' ) : 'info' %>
```

Comment:
Could this be better organised as a multiline conditional instead of a lambda inside a lambda?